src/Components/App/App.js

```diff
@@ -1,13 +1,37 @@
-import React from 'react';
+import React, {useState} from 'react';
 import Dashboard from '../Dashboard/Dashboard.js';
 import './App.css';
 import LandingPage from '../LandingPage/LandingPage'
 import Footer from '../Footer/Footer'
+import {fetchUserLocation, fetchInputLocation} from '../../utilities'
+
 
 function App() {
+  const [location, setLocation] = useState({});
+  const [dashboardView, setDashboardView] = useState(false);
+
+  const grabUserLocationData = (event) => {
```

Comment:
This looks really great. I'm wondering if there is a way we can eventually refactor this function and the grabInputLocationData into a dynamic one. I am seeing that grabInputLocationData is taking in parameters and this one is not so that could be tricky, but I think with arrow functions there is a way it can be done. Something to explore eventually